index.html

```diff
@@ -32,15 +30,18 @@ <h1>
       <div id="insert">
         <p>Inserisci l'email associata all'account</p>
       </div>
-      <div id="forms">
+      <div class="forms">
```

Comment:
```suggestion
      <div id="forms">
```
chyba nie chciałeś zmienić z id na class

Reply:
okej rzeczywiscie, poprawiam